Scripts/xbbtools/nextorf.py

```diff
@@ -220,7 +214,7 @@ def Output(self, CDS):
                     n - start + 1,
                 )
             if self.options["gc"]:
-                head = "%s:%s" % (head, self.Gc2(subs))
+                head = f"{head}:{self.Gc2(subs)}"
```

Comment:
It is this kind of f-string which I find most jarring - single variable substitutions I'm absolutely happy with, actual code far less so. I think my editor needs a syntax colouring update which would help.